contracts/zero-ex/tests/utils/TestUtils.sol

```diff
@@ -29,4 +36,126 @@ contract TestUtils is Test {
             }
         }
     }
+
+    function getSigner() public returns (address, uint) {
+        string memory mnemonic = "test test test test test test test test test test test junk";
+        uint256 privateKey = vm.deriveKey(mnemonic, 0);
+        vm.label(vm.addr(privateKey), "zeroEx/MarketMaker");
+        return (vm.addr(privateKey), privateKey);
+    }
+
+    function makeTestRfqOrder(
+        DeployZeroEx.ZeroExDeployed memory zeroExDeployed, 
+        IERC20Token makerToken, 
+        IERC20Token takerToken,
+        address makerAddress,
+        address takerAddress,
+        uint256 makerKey
+    ) public returns (bytes memory callData) {
```

Comment:
Generally, tests shouldn't have `public`/`external` functions that aren't test cases